src/Rules/Doctrine/ORM/EntityColumnRule.php

```diff
@@ -149,8 +149,8 @@ public function processNode(Node $node, Scope $scope): array
 				'Property %s::$%s type mapping mismatch: database can contain %s but property expects %s.',
 				$className,
 				$propertyName,
-				$writableToPropertyType->describe(VerbosityLevel::typeOnly()),
-				$property->getWritableType()->describe(VerbosityLevel::typeOnly())
+				$writableToPropertyType->describe(VerbosityLevel::getRecommendedLevelByType($writableToPropertyType)),
+				$property->getWritableType()->describe(VerbosityLevel::getRecommendedLevelByType($propertyWritableType))
```

Comment:
Both levels should be determined like this: `VerbosityLevel::getRecommendedLevelByType($propertyWritableType, $writableToPropertyType);`